hack/check

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export CGO_ENABLED=0
+
+find=find
+
+if [ "$(uname)" == "Darwin" ]; then
+	if ! command -v gfind ; then
+		echo "please install GNU Find"
+		echo ""
+		echo "    brew install findutils"
+		exit 1
+	fi
+	find=gfind
+fi
```

Comment:
I think we can omit this code and use `PKGS=$(find pkg -type f -name '*.go'|awk '{print "github.com/pingcap/tidb-operator/"$1}'|sort -u)` instead.